app/src/release/res/values/google_maps_api.xml

```diff
@@ -17,6 +17,6 @@
     string in this file.
     -->
     <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">
-        YOUR_KEY_HERE
+        AIzaSyCmPIFTlPKolQVgLNnQh8TseBzW9FAD8ig
```

Comment:
You added the key here and still you don't use it in the manifest. How come?

Reply:
I first inserted @string/google_maps_key
but when I did so, Google was still convinced the value of the field of this reference was "YOUR_KEY_HERE", even though it was updated already.
It only worked for me by pasting the Key directly into the manifest.

Reply:
I see, let's remove this key here then.